core/log/logger.go

```diff
@@ -0,0 +1,9 @@
+package log
+
+type Logger interface {
```

Comment:
Should we keep the "Service" nomenclature?

Reply:
yea that is cleaner